src/styles/_header.scss

```diff
@@ -0,0 +1,81 @@
+$link-color: #00acdc;
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  width: 100%;
+  padding-right: 50px;
+  padding-left: 50px;
+  display: flex;
+
+  &__image {
+    padding-top: 4px;
+    text-decoration: none;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    align-items: center;
+    margin: 0;
+  }
+
+  &__item {
+    list-style: none;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    font-family: "Roboto", sans-serif;
+    font-weight: 500;
+    position: relative;
+    text-decoration: none;
+    color: #060b35;
```

Comment:
please, try to organize your styles
I recommend you take a look at these styles code and use it in the feature 
[9elements](https://9elements.com/css-rule-order) or  [htmlAcademy](https://codeguide.academy/html-css.html#css-order)